### PR TITLE
feat: make-payment

### PR DESCRIPTION
**Pull request by EsamSharaf**
adding make-payment visualization

### CODE REVIEW

File: WebCoffeeMachine/coffeemachine/templates/coffeemachine/detail.html [hunk 1]
```diff
@@ -6,32 +6,37 @@
...
             </div>
+            <div class="detail-block-2">
+                {{item.description}}
+            </div>
         </div>
-        <div class="detail-block-2">
-            {{item.description}}
+        <div id="overlay">
+            <h2>Please make the payment</h2>
+            <img src="../../static/coffeemachine/images/cashless_payment-transparent.png" alt="alt-image">
```
Comment (blancheta) [inline WebCoffeeMachine/coffeemachine/templates/coffeemachine/detail.html:38]: ../.. should not be necessary here

File: WebCoffeeMachine/coffeemachine/static/coffeemachine/style.css [hunk 2]
```diff
@@ -193,4 +193,32 @@ h3 {
...
-}
\ No newline at end of file
+}
+
+#container {
+	opacity: 1;
+}
+
+#overlay {
+	position: fixed;
```
Comment (blancheta) [inline WebCoffeeMachine/coffeemachine/static/coffeemachine/style.css:203]: no need for 8 spaces here, 4 spaces for the indentation should be enough

File: WebCoffeeMachine/coffeemachine/templates/coffeemachine/detail.html [hunk 3]
```diff
@@ -6,32 +6,37 @@
...
-                    <div class="row-3">
-                        <p id="before-total-text">TOTAL:&#65129;<span id="total">{{item.price}}</span></p>
-                    </div>
-                    
-                    <div class="row-4">
-                        <a href='' target="_self">PAY NOW</a>
+        <div id="container">
+            <h1>Your Selection</h1>
+            <div class="detail-block-1">
+                <div class="detail-block-1-1">
```
Comment (blancheta) [inline WebCoffeeMachine/coffeemachine/templates/coffeemachine/detail.html:12]: can you choose a more meaningful name here "detail-block-1-1"?

File: WebCoffeeMachine/coffeemachine/static/coffeemachine/style.css (hunk 2 above)
Comment (EsamSharaf) [reply WebCoffeeMachine/coffeemachine/static/coffeemachine/style.css:203]: Lines like line 203 indented by  1 tab (4spaces). Could you elaborate?

File: WebCoffeeMachine/coffeemachine/templates/coffeemachine/detail.html (hunk 1 above)
Comment (EsamSharaf) [reply WebCoffeeMachine/coffeemachine/templates/coffeemachine/detail.html]: Changed in commit # [5bbe665](https://github.com/EsamSharaf/WebCoffeeMachine/pull/5/commits/5bbe66577be561bd2edcc364876a1bca47fa1838)

File: WebCoffeeMachine/coffeemachine/templates/coffeemachine/detail.html (hunk 3 above)
Comment (EsamSharaf) [reply WebCoffeeMachine/coffeemachine/templates/coffeemachine/detail.html]: the noted reflected in commit # [466c2aa](https://github.com/EsamSharaf/WebCoffeeMachine/pull/5/commits/466c2aa9ec6dcae7b9a799df5b0e12fde64078b9)

File: WebCoffeeMachine/coffeemachine/static/coffeemachine/style.css (hunk 2 above)
Comment (blancheta) [reply WebCoffeeMachine/coffeemachine/static/coffeemachine/style.css:203]: Then that's the preview on github. good

File: WebCoffeeMachine/coffeemachine/static/coffeemachine/style.css [hunk 4]
```diff
@@ -193,4 +193,32 @@ h3 {
...
+	color: white;
+	font-family: "Times New Roman", Helvetica, sans-serif;
+}
+
+#overlay img {
+	width: 50%;
+	height: 50%;
+}
+
+#overlay { animation: fadeIn 1s; }
```
Comment (blancheta) [inline WebCoffeeMachine/coffeemachine/static/coffeemachine/style.css:219]: i would put overlay before overlay img

**Comment by blancheta** [comment]
Just a minor change to make and we are good

File: WebCoffeeMachine/coffeemachine/static/coffeemachine/style.css (hunk 4 above)
Comment (EsamSharaf) [reply WebCoffeeMachine/coffeemachine/static/coffeemachine/style.css]: The adjustment made in commit # [e2bf76a](https://github.com/EsamSharaf/WebCoffeeMachine/pull/5/commits/e2bf76aebbff2f2efcba382ba26999c19d7dfa4c)

**Comment by blancheta** [comment]
Good

File: WebCoffeeMachine/coffeemachine/static/coffeemachine/style.css [hunk 5]
```diff
@@ -73,18 +73,18 @@ a {
 h3 {
 	color: white;
 }
```
Comment (blancheta) [inline WebCoffeeMachine/coffeemachine/static/coffeemachine/style.css:75]: Let´s go!

**Review by blancheta — APPROVED** [review]

**Pull request merged**